lisp/forge-commands.el

```diff
@@ -67,7 +67,12 @@ for a repository using the command `forge-add-pullreq-refspec'."
     ("c f" "fork or remote" forge-fork)]]
   [["Configure"
     ("a" "add repository to database" forge-add-repository)
-    ("r" "forge.repository" forge-forge.remote)]])
+    ("r" "forge.repository" forge-forge.remote)
+    ("s" (lambda ()
+           (if forge-display-sections
+               "hide PRs/issues"
+             "display PRs/issues")) forge-toggle-display-sections)]])
+
```

Comment:
Generally avoid following a multi-line s-expression by another s-expression without a newline character in between.